sar/examples/S18E047_2021_01_08_04392F.xml

```diff
@@ -29,7 +29,7 @@
             <OrbitDataSource>RESORB</OrbitDataSource>
             <OrbitDataFileName>S1A_OPER_AUX_RESORB_OPOD_20210108T062130_V20210108T012450_20210108T044220.EOF</OrbitDataFileName>
             <PlatformHeading units="deg">-167.7482429286898</PlatformHeading>
-            <OrbitMeanAltitude units="km">693</OrbitMeanAltitude>
+            <OrbitMeanAltitude units="m">693000</OrbitMeanAltitude>
```

Comment:
fyi: This will get reverted once we update the file to 5.5 from an external source.